test/portfolio_test_lib.go

```diff
@@ -260,22 +232,16 @@ func StEq(
 	}
 }
 
+// TODO: remove
 func SoftSflAlmostEqual(t *testing.T, expDelta TDt, delta *ptf.TxDelta) bool {
```

Comment:
Only in a couple places, so go ahead I guess?

Reply:
Removed `SoftStEq` too. Replaced directly by `require.Equal` which I think is correct